builder/azure/arm/config.go

```diff
@@ -571,7 +578,10 @@ func (c *Config) createCertificate() (string, error) {
 		err = fmt.Errorf("Failed to Generate Private Key: %s", err)
 		return "", err
 	}
+	return c.formatCertificateForKeyVault(privateKey)
+}
 
+func (c *Config) formatCertificateForKeyVault(privateKey *rsa.PrivateKey) (string, error) {
 	host := fmt.Sprintf("%s.cloudapp.net", c.tmpComputeName)
 	notBefore := time.Now()
```

Comment:
i know this is outside the scope of this PR, but I feel like the notBefore should be time.Now - 5mins

Reply:
to allow to machine time skews. pls see if you can incorporate this

Reply:
I think this is more related to the SSH PR than this one, which has merged.

Given the boot time of a Windows VM, I think we've got a bit of leeway in here already, so at this point, I think we should see how it plays out, and come back to it if need be.